spyder/plugins/editor/panels/codefolding.py

```diff
@@ -239,6 +240,14 @@ def paintEvent(self, event):
         # on the folding panel.
         super(FoldingPanel, self).paintEvent(event)
         painter = QPainter(self)
+        cell_line_color = QColor(Qt.darkGray)
```

Comment:
Let set this as a class attribute like the one in the linenumber panel at: https://github.com/spyder-ide/spyder/pull/20509/files#diff-04bc4dbe5c8af45aeb3dc9d7aef7317a8ae81284b422f34c5e6447c233d54324R37